src/arrays.jl

```diff
@@ -42,19 +42,20 @@ function deepdiff(X::Vector, Y::Vector)
     removed = Int[]
     added = Int[]
 
-    backtrack(backtracks, removed, added, length(X)+1, length(Y)+1)
+    backtrack(backtracks, removed, added, (length(X)+1, length(Y)+1))
 
     VectorDiff(X, Y, removed, added)
 end
 
 # recursively trace back the longest common subsequence, adding items
 # to the added and removed lists as we go
-function backtrack(backtracks, removed, added, i, j)
-    bt = backtracks[i, j]
+function backtrack(backtracks, removed, added, ij)
+    bt = backtracks[ij...]
     if bt != (0, 0)
-        backtrack(backtracks, removed, added, ((i, j) .- bt)...)
+        backtrack(backtracks, removed, added, ij .- bt)
     end
 
+    (i, j) = ij
```

Comment:
Probably better to do this earlier and have:
```julia
i, j = ij
bt = backtracks[i, j]
```

Reply:
better I think to just keep `i` and `j` as separate arguments to `backtrack`, then just have `bti, btj = backtracks[i,j]`.